presto-main/src/test/java/com/facebook/presto/type/TestMapOperators.java

```diff
@@ -442,22 +442,35 @@ public void testJsonToMap()
     public void testElementAt()
             throws Exception
     {
+        // empty map
```

Comment:
Is every case that existed before covered by the new tests? I looked briefly and it seemed that some of them have been removed (?)